web-components/src/utils/locationStandard.test.ts

```diff
@@ -0,0 +1,82 @@
+import { getJurisdictionIsoCode } from './locationStandard';
```

Comment:
We can consider making `.spec` our convention - that is what I see most commonly. But `.test` is fine too.

Reply:
Thanks so much for setting up tests!

Reply:
I don't have a strong preference, both are convention. Currently the project is using `.test.` , but they are only a few cases. Let's talk about it at the next team meeting.